api/db.json

```diff
@@ -1,4 +1,317 @@
+              }
+            }
+          }
+        },
+        {
+          "name": "huskyCI",
+          "website": "https://huskyci.opensource.globo.com/",
+          "repositoryUrl": "https://github.com/globocom/huskyCI",
+          "featured": true,
+          "imageUrl": "https://opensource.globo.com/static/huskici-79ad4c8ec6669c1469abb7d059e3786a.svg",
+          "documentationUrl": "https://huskyci.opensource.globo.com/docs/quickstart/overview",
+          "description": "Performing security tests inside your CI.",
+          "stats": {
+            "repository": {
+              "object": {
+                "commit": {
+                  "history": {
+                    "totalCount": 1006
+                  }
+                }
+              },
+              "issues": {
+                "totalCount": 169
+              },
+              "pullRequests": {
+                "totalCount": 350
+              },
+              "stars": {
+                "totalCount": 352
+              }
+            }
+          }
+        },
+        {
+          "name": "Megadraft",
+          "website": "http://megadraft.io/",
+          "repositoryUrl": "https://github.com/globocom/megadraft",
+          "featured": true,
+          "imageUrl": "https://opensource.globo.com/static/megadraft-f0be51efa0afd965dc252addc1cd46ff.svg",
+          "documentationUrl": "http://megadraft.io/#/docs/overview",
+          "description": "Megadraft is a Rich Text editor built on top of Facebook's Draft.JS featuring a nice default base of components and extensibility.",
+          "stats": {
+            "repository": {
+              "object": {
+                "commit": {
+                  "history": {
+                    "totalCount": 579
+                  }
+                }
+              },
+              "issues": {
+                "totalCount": 144
+              },
+              "pullRequests": {
+                "totalCount": 184
+              },
+              "stars": {
+                "totalCount": 949
+              }
+            }
+          }
+        },
+        {
+          "name": "Fastlane Queue",
+          "website": "",
+          "repositoryUrl": "https://github.com/fastlane-queue",
+          "featured": false,
+          "documentationUrl": "https://fastlane.readthedocs.io/",
+          "description": "More seriously, though, fastlane allows you to easily implement new workers in the form of containers."
+        },
+        {
+          "name": "secDevLabs",
+          "website": "",
+          "repositoryUrl": "https://github.com/globocom/secDevLabs",
+          "featured": true,
+          "imageUrl": "https://raw.githubusercontent.com/globocom/secDevLabs/master/images/secDevLabs-logo.png",
+          "documentationUrl": "https://github.com/globocom/secDevLabs/blob/master/README.md",
+          "description": " laboratory for learning secure web and mobile development in a practical manner.",
```

Comment:
It looks like an `A laboratory...` is missing here! 😅

Reply:
this is the same API output: https://opensource.globo.com/edition

So, I will fix both ;) Thanks!